entry_timeseries_test.go

```diff
@@ -66,18 +78,34 @@ var _ = Describe("Tests", func() {
 				}
 			})
 			JustBeforeEach(func() {
-				err := timeseries.Create()
+				err := timeseries.Create(columnsInfo...)
 				Expect(err).ToNot(HaveOccurred())
-				Expect(columnsInfo[0].Type()).To(Equal(timeseries.ColumnInfos()[0].Type()))
-				Expect(columnsInfo).To(ConsistOf(timeseries.ColumnInfos()))
+				doubleColumn = timeseries.DoubleColumn(columnsInfo[1].Name())
+				blobColumn = timeseries.BlobColumn(columnsInfo[0].Name())
+			})
+			It("should have one blob column and one double column", func() {
```

Comment:
Add `It("should work when timeseries and columns are already created")` test case.
Something like:
```
anotherTimeseries = handle.Timeseries(alias)
anotherDoubleColumn = anotherTimeseries.DoubleColumn(columnsInfo[1].Name())
anotherDoubleColumn.Insert(...)
```